### DISCUSSION TITLE
Is there a simple solution with LiteLLM for web search + LLM?

### DISCUSSION THREAD

**Discussion by rosmur** [BerriAI/litellm#4363] [Q&A]
Basically looking for something as clean/simple as Cohere's web-search connector. But generalizable to use Claude Sonnet etc.

**Comment by ghost** [comment]
hey @rosmur can you share the relevant cohere docs on this?

**Reply by rosmur** [reply]
Cant seem to find it in the official docs, but here's an example: https://github.com/FilterJoe/CohereAPI/blob/master/web_test.py. Its just 1 additional line: `connectors=[{"id": "web-search",
    			 "user_access_token": api_key}`

**Reply by rosmur** [reply]
If such functionality is enabled within LiteLLM (with say a SERP API if needed), y'all will be GOATed. 

I have no idea why Web Search enabled LLM isnt the mainstream/standard solution already. Static LLMs are cool but not actually good for any real purpose. You cant ever 100% trust it.
